core/src/main/java/com/linecorp/armeria/server/HttpHeaderUtil.java

```diff
@@ -128,7 +128,7 @@ static void getAllValidAddress(@Nullable String headerValue,
                     accumulator.add(addr);
                 }
             } catch (UnknownHostException e) {
-                logger.debug("Failed to resolve hostname: {}", v, e);
+                logger.debug("Failed to create an IP address: {}", v, e);
```

Comment:
Failed to create an InetSocketAddress? Obviously you did not create an IP address.